pkg/registry/file/containerprofile_user_managed.go

```
@@ -29,16 +29,22 @@ const (
-	// the ug- AP / NN that contributed to this merge. Absent annotation means
-	// no ug- of that kind was present at merge time.
+	// mergedSourceUserCPKey records the storage key of the ug- ContainerProfile
+	// that contributed to this merge. Absent annotation means no ug- CP was
+	// present at merge time.
+	mergedSourceUserCPKey = "kubescape.io/merged-source-ug-cp"
+
+	// Deprecated: the ug- overlay is now a single ContainerProfile; use
+	// mergedSourceUserCPKey. mergedSourceUserAPKey / mergedSourceUserNNKey recorded
+	// the storage keys of the ug- AP / NN that contributed to the merge under the
+	// legacy AP+NN overlay model. Retained for backward-compatibility views.
 	mergedSourceUserAPKey = "kubescape.io/merged-source-ug-ap"
```

golangci-lint (2.12.2)

[error] const mergedSourceUserAPKey is unused

(unused)